mod/wikirate/spec/set/right/wikirate_article_spec.rb

```diff
@@ -1,41 +1,49 @@
+      Card::Env.params[:citable] = @claim.name
+      @citation = 'Death Star uses dark side of the Force '\
+                  '{{Death Star uses dark side of the Force|cite}}'
     end
-    context "missing view" do
-      it "render editor with empty content with citation tips" do
-        article = Card.new :name=>"#{@company.name}+#{@topic.name}+#{Card[:wikirate_article].name}",:type=>"basic"
+    context 'missing view' do
+      it 'render editor with empty content with citation tips' do
+        name = "#{@company.name}+#{@topic.name}+#{Card[:wikirate_article].name}"
+        article = Card.new name: name, type: 'basic'
         html = article.format.render_missing
-
-        expect(html).to have_tag("div",:with=>{:class=>"claim-tip"}) do
-          with_tag "textarea",:with=>{:id=>"citable_claim"},:text=>/#{@citation}/
+        id = 'citable_note'
+        expect(html).to have_tag('div', with: { class: 'note-tip' }) do
+          with_tag 'textarea', with: { id: id }, text: /#{@citation}/
         end
-        expect(html).to have_tag("textarea",:with=>{:class=>"tinymce-textarea",:name=>"card[content]"})
-
+        expect(html).to have_tag('textarea', with: { class: 'tinymce-textarea',
+                                                     name: 'card[content]' })
       end
     end
-    context "content and titled_with_edits views" do
-      it "render editor with content with citation tips" do
-        overview_name = "#{@company.name}+#{@topic.name}+#{Card[:wikirate_article].name}"
-        article = Card.create :name=>overview_name, :type=>"basic",:content=>"hello world"
+    context 'content and titled_with_edits views' do
+      it 'render editor with content with citation tips' do
+        overview_name = "#{@company.name}+#{@topic.name}+"\
+                        "#{Card[:wikirate_article].name}"
+        article = Card.create name: overview_name, type: 'basic',
+                              content: 'hello world'
         html = article.format.render_content
-        expect(html).to have_tag("div",:with=>{:class=>"claim-tip"}) do
-          with_tag "textarea",:with=>{:id=>"citable_claim"},:text=>/#{@citation}/
+        id = 'citable_note'
+        expect(html).to have_tag('div', with: { class: 'note-tip' }) do
+          with_tag 'textarea', with: { id: id }, text: /#{@citation}/
         end
-        expect(html).to have_tag("textarea",:with=>{:class=>"tinymce-textarea",:name=>"card[content]"},:text=>/hello world/)
+        expect(html).to have_tag('textarea', with: { class: 'tinymce-textarea',
+                                                     name: 'card[content]' },
+                                             text: /hello world/)
 
         html = article.format.render_titled_with_edits
-        expect(html).to have_tag("div",:with=>{:class=>"claim-tip"}) do
-          with_tag "textarea",:with=>{:id=>"citable_claim"},:text=>/#{@citation}/
+        expect(html).to have_tag('div', with: { class: 'note-tip' }) do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.